packages/v4/patternfly-docs/content/design-guidelines/components/menu/menu.md

```diff
@@ -5,7 +5,22 @@ section: components
 
 A **menu** is a list of options or actions that users can choose from. It can be used in a variety of contexts whenever the user needs to choose between multiple values, options, or actions. A menu can be opened in a [dropdown](/components/dropdown) or [select list](/components/select), or it can be revealed by right clicking on a specific region within a page.
 
-## Elements and variations
+## Elements
+Elements vary depending on [menu variation](#Variations)
+
+## Usage
+
+| **When you want to...** |                     |
+| ------------------- | ------------------- |
+| Expose a list of actions | Use an actions menu with actions and/or links. Add icons in front of menu items only if they will help to make items more recognizable. Include descriptive text for menu items that may require further definition. |
+| Create logical groupings of menu items | Use separators to group items. Include group headings if you don’t feel like the meaning of the groups will be obvious. |
+| Expose a list of nested actions | Use a fly-out menu. Any optional attributes of basic menus, including icons, groups, and descriptions can also be applied to flyout menus.|
+| Select from a list of options | Use an options select menu for selecting between values rather than actions. Icons, descriptions, and groupings can be used with option select menus to improve recognition or create logical option groups.
+| 
```

Comment:
Make sure this line is on the same line as the row itself (ie: just backspace 1)